cli/cmd/plugin/standalone-cluster/tanzu/tanzu.go

```diff
@@ -32,7 +32,7 @@ const (
 	configFileName        = "config.yaml"
 	tanzuConfigDir        = "tanzu"
 	tkgConfigDir          = "tkg"
-	localConfigDir        = "local"
+	standaloneConfigDir   = "standalone"
```

Comment:
Note: This PR also introduces a _breaking change_ where the configuration directory changes from

```
~/.config/tanzu/tkg/local/...
```

to

```
~/.config/tanzu/tkg/standalone/...
```

creating clusters and everything still works the same, but users won't be able to delete / list clusters they have currently in the `local` directory